src/legacy/server/index_patterns/service/lib/field_capabilities/field_caps_response.js

```diff
@@ -120,7 +120,26 @@ export function readFieldCapsResponse(fieldCapsResponse) {
       aggregatable: isAggregatable,
       readFromDocValues: shouldReadFieldFromDocValues(isAggregatable, esType),
     };
-  }).filter(field => {
+  });
+
+  // Get all types of sub fields. These could be multi fields or children of nested/object types
+  const subFields = kibanaFormattedCaps.filter(field => {
+    return field.name.includes('.');
+  });
+
+  // Discern which sub fields are multi fields. If the parent field is not an object or nested field
+  // the child must be a multi field.
+  subFields.forEach(field => {
+    const parentFieldName = field.name.split('.').slice(0, -1).join('.');
+    const parentFieldCaps = kibanaFormattedCaps.find(caps => caps.name === parentFieldName);
+
+    if (!['object', 'nested'].includes(parentFieldCaps.type)) {
+      field.parent = parentFieldName;
+      field.subType = 'multi';
```

Comment:
Why not just have a `isMulti` boolean instead of adding another string property `subType`? What else could `subType` be?

Reply:
Do we even need the `multi` designation? Can we just assume if `parent` is not undefined, it's multi?

Reply:
In the future it could be `nested`, `object`, or `join`. I see this being used to describe any parent/child relationship between fields. I thought about just doing an `isMulti` flag but then we'd need to add `isNested`, `isObject` and `isJoin` in the future.

Instead of having a separate parent field and a flag we could also just have something like `multiParent`, `nestedParent`, `objectParent`, `joinParent`. We'd still have to add a new one for each new relationship type, but then they're not coupled together. 

I dunno, what do you think?

Reply:
I've made all the updates I can without making a decision on this. Once we've agreed on a format I'll go through and update all the other places mentioned that need these properties.